### PR TITLE
Update distribution doc for IBM IKS

### PR DESCRIPTION
**Pull request by yhwang**
- update the deployment instructions to use v1.7-branch
- update the OWNERS files to reflect the ownership changes

### CODE REVIEW

File: content/en/docs/distributions/ibm/deploy/OWNERS [hunk 1]
```diff
@@ -1,7 +1,5 @@
 approvers:
-  - animeshsingh
```
Comment (Tomcli) [inline content/en/docs/distributions/ibm/deploy/OWNERS:2]: Hi @yhwang, maybe we can do the owner update in another PR. Animesh and shawn no long work at IBM so I can promote you as the new approver as well.

Comment (yhwang) [reply content/en/docs/distributions/ibm/deploy/OWNERS:2]: sure, let me remove the changes for the OWNERS file in this PR for now

**Pull request merged**